addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -36,6 +40,20 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _launch_session: function () {
+        this._rpc({
+            'model': 'survey.survey',
+            'method': 'action_start_session',
+            'args': [[parseInt(this.$('.o_survey_session_launch_session').attr('survey_id'))]],
```

Comment:
To store data on blocks we use "data" attributes.

So in your template you will have:
`t-att-data-survey-id="survey_id"` 
And in the Javascript:
`this.$('.o_survey_session_launch_session').data('surveyId')`

(It will magically convert "survey-id" to "surveyId").